tests/pipeline/test_dataset.py

```diff
@@ -23,6 +24,10 @@ class LargeDataSet(DataSet):
     })
 
 
+class InvalidColumnDataset(DataSet):
+    currency_aware_date = Column(dtype=datetime64ns_dtype, currency_aware=True)
```

Comment:
Talked about this in person, but I think we want this statement to error rather than the attribute access.